packages/langium/src/lsp/completion/completion-provider.ts

```diff
@@ -239,6 +239,10 @@ export class DefaultCompletionProvider implements CompletionProvider {
     }
 
     protected completionForKeyword(keyword: ast.Keyword, context: AstNode | undefined, acceptor: CompletionAcceptor): MaybePromise<void> {
+        // Filter out keywords that do not contain any word character
+        if (!keyword.value.match(/[\w]+/)) {
+            return Promise.resolve();
```

Comment:
I think you can just return nothing here because `MaybePromise` does not force you to create a Promise.

Reply:
Changed